src/components/templates/Jazzbar/components/JazzBarRoom/Room.scss

```diff
@@ -0,0 +1,14 @@
+@import "scss/constants.scss";
```

Comment:
This file seems to duplicate our generic components styles from src/components/molecules/TableComponent/TableComponent.scss

What's the reasoning/need for duplicating it?

Reply:
It seems the following are the only differences:

```diff
diff --git a/src/components/organisms/Room/Room.scss b/src/components/templates/Jazzbar/components/JazzBarRoom/Room.scss
index 535b710f..3c4bbd99 100644
--- a/src/components/organisms/Room/Room.scss
+++ b/src/components/templates/Jazzbar/components/JazzBarRoom/Room.scss
@@ -1,18 +1,10 @@
 @import "scss/constants.scss";
 
-.participant-container {
+.jazzbar-participant-container {
   background: $black;
   display: flex;
   justify-content: center;
   width: 100%;
-  align-self: flex-start;
-
-  &.two-across {
-    grid-column: span 3;
-  }
-  &.three-across {
-    grid-column: span 2;
-  }
 }
 
 .empty-chair-image {
```